variables.tf

```diff
@@ -27,6 +27,11 @@ variable "region" {
 # ECS instance
 ##################
 
+variable "create_rds_mysql" {
+  description = "Whether to build WordPress through RDS MySQL module. If it is true use RDS MySQL module. Otherwise install MySQL on ECS."
```

Comment:
description = "Whether to create RDS MySql database. If false, this module will create MySql database on ECS."